scene/open-mobile-scene-package/README.md

```diff
@@ -1,26 +1,30 @@
 # Open mobile scene package
 
-Open and display a scene from an offline Mobile Scene Package (.mspk).
+Opens and displays a scene from a Mobile Scene Package (.mspk).
 
-![](OpenMobileScenePackage.png)
+![Image of open mobile scene package](OpenMobileScenePackage.png)
 
-## Use Case
+## Use case
 
-An .mspk file is an archive containing the data (specifically, basemaps and features) used to display an offline 3D scene.
+A mobile scene package is an archive containing the data (specifically, basemaps and features), used to display an offline 3D scene.
 
 ## How to use the sample
 
-The sample loads a mobile scene package. Pan and zoom to explore the scene.
+When the sample opens, it will automatically display the Scene in the Mobile Scene Package.
```

Comment:
```suggestion
When the sample opens, it will automatically display the Scene in the mobile scene package.
```

Reply:
I'll take your suggestion here, but I do feel like it should be capitalized because it's a named term?